### PR TITLE
Attempt to wire up Brushless settings

### PR DESCRIPTION
**Pull request by jedahan**

### CODE REVIEW

File: src/server.ts [hunk 1]
```diff
@@ -221,7 +222,7 @@ export function startServer(port: number, device: string | null = null, enableCo
   return new Promise((resolve) => {
     server.listen(port, () => {
       async function connect() {
-        for await (const d of ebbs(device)) {
+        for await (const d of ebbs(device, model)) {
           ebb = d;
           broadcast({c: "dev", p: {path: ebb ? /*ebb.port.path*/"/dev/XXX" : null}});
         }
```
Comment (jedahan) [inline src/server.ts:228]: this seems fragile - all EBBs need to be of the same model :/

**Comment by alexrudd2** [comment]
Oh, I'm sorry I didn't see this!  I was working locally and ended up doing many of the same things, then pushed them last night.  Let me take a look at what you've done.  Thanks for the PR!

**Comment by jedahan** [comment]
It's taking me a bit of time to test since I'm on a pi zero and need to compile a modern nodejs first - will undraft after testing

**Comment by alexrudd2** [comment]
OK, so it looks like you and I ended up basically doing the same thing.  Slightly different terminology but the same code patterns.

**Comment by alexrudd2** [comment]
I'm going to rebase on top of `main` to reduce some of the conflicts.

**Comment by jedahan** [comment]
i have to split my changes into 2 branches one that is brushless specific, and one that is raspberry pi zero specific.

**Comment by jedahan** [comment]
FYI this old patchset is working fine on my axidraw with brushless upgrade - lmk if you want me to change anything or abandon it or whatever. Its gotten a bit messy so I can rebase and re-apply, or wait for your changes to be ready to test.

**Comment by alexrudd2** [comment]
> FYI this old patchset is working fine on my axidraw with brushless upgrade - lmk if you want me to change anything or abandon it or whatever. Its gotten a bit messy so I can rebase and re-apply, or wait for your changes to be ready to test.


What do you mean by "old patchset" - my branch, or yours?  I don't particularly care which implementation we use.

Mine works on real hardware with one exception - it smashes the pen to the limit stop at the end of the drawing.  What numbers are you using here?
https://github.com/alexrudd2/saxi/blob/f061831179ba0d5bcc950f6a17ad55a3c435e38c/src/planning.ts#L120-L121

**Comment by jedahan** [comment]
Oh I just meant that the changes in `jedahan:brushless` diverged from its original purpose of just wiring up what was there. I'll try using `alexrudd2:brushless` and if that obsoletes jedahan:brushless-v2 will close both branches.

**Comment by jedahan** [comment]
I was using 

```
  penServoMin: 5400,  // pen down
  penServoMax: 9600, // pen up
```

as you were, but https://github.com/nornagon/saxi/issues/136 says it should be 

```
        penServoMin: 5400,
        penServoMax: 12600,
```

or with the strong spring:
```
        penServoMin: 5400,
        penServoMax: 10000,
```

**Pull request closed without merging**